src/main/java/com/dev/cinema/dao/impl/AbstractDao.java

```diff
@@ -34,6 +39,9 @@ protected T add(T instance, Class<T> clazz) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             Query<T> query = session.createQuery("from " + clazz.getSimpleName(), clazz);
             return query.getResultList();
+        } catch (Exception e) {
+            logger.error("Cant fetch all {} from DB", clazz.getName(), e);
```

Comment:
```suggestion
            logger.error("Can't create the {} instance", clazz.getName(), e);
```

Reply:
I wouldn't understand what is this message about :)
Let's fix the message for all of the _**add**_ methods

Reply:
This is getAll() method. I don't do logging in add() because of throwing exception.